detector_test.go

```diff
@@ -25,9 +24,9 @@ func TestDetector(t *testing.T) {
 func testDetector(t *testing.T, when spec.G, it spec.S) {
 	var (
 		config      *lifecycle.DetectConfig
-		outLog      *bytes.Buffer
 		platformDir string
 		tmpDir      string
+		logHandler  = &memory.Handler{}
```

Comment:
Things in this context are executed many times and not shared across all `it`s. Was this intentional?

Reply:
I don't believe we want to share the same `logHandler` across all `it`s

Reply:
Fixed